src/compas_xr/assemblies/assembly_extensions.py

```diff
@@ -0,0 +1,80 @@
+import os
+# from Rhino.Geometry import Mesh, MeshingParameters
+from compas.datastructures import Assembly, Mesh
+from compas.geometry import Transformation, Point, Vector, Frame
+from compas_timber import assembly as TA
+from compas_timber.consumers import BrepGeometryConsumer
+
+class AssemblyExtensions(object):
+
+    def __init__(self):
+        pass
```

Comment:
Delete, There's no need for an empty default init.

```suggestion
    def __init__(self):
        pass
```

Reply:
Done :)